open_ce/build_image.py

```diff
@@ -84,10 +84,14 @@ def build_runtime_container_image(args):
         conda_env_file = os.path.abspath(conda_env_file)
         _validate_input_paths(local_conda_channel, conda_env_file)
 
-        # Copy the conda environment file into the local conda channel to modify it
-        shutil.copy(conda_env_file, local_conda_channel)
-        conda_env_file = os.path.join(local_conda_channel, os.path.basename(conda_env_file))
-        utils.replace_conda_env_channels(conda_env_file, r'file:.*', "file:/{}".format(TARGET_DIR))
+        # Copy the conda environment file into the local conda channel with a new name and modify it
+        conda_env_runtime_filename = os.path.splitext(os.path.basename(args.conda_env_file))[0]+'-runtime.yaml'
```

Comment:
This should be args.conda_env_files
```suggestion
        conda_env_runtime_filename = os.path.splitext(os.path.basename(args.conda_env_files))[0]+'-runtime.yaml'
```

Reply:
yeah thanks. I had to merge an update in this one yet again and I used the web merge this time while on the phone. Not recommended. :) I'll fix those.

Reply:
actually this should just be
```
conda_env_runtime_filename = os.path.splitext(os.path.basename(conda_env_files))[0]+'-runtime.yaml'
```
since we are working with one of the env files in the list at this point.